src/components/date-picker/flatpickr-adapter/flatpickr-adapter.tsx

```diff
@@ -14,6 +14,8 @@ import { YearPicker } from '../pickers/YearPicker';
  * This component is internal and only supposed to be used by
  * the limel-date-picker. This component is needed in order for us
  * to render the flatpickr calendar in a portal.
+ *
+ * @private
```

Comment:
Ah! Nice! I think I recognise this, so you've probably told me at point, and I had just forgotten it was possible. Great!